src/shared/utils/storage.ts

```diff
@@ -5,6 +5,8 @@ export const enum LocalStorageKeys {
   LibraryPreparedData = 'libraryPreparedData',
   IsFromLibrary = 'isFromLibrary',
   LibraryUrl = 'libraryPath',
+  EHRBannerAvailableDismissed = 'EHRBannerAvailableDismissed',
+  EHRBannerActiveDismissed = 'EHRBannerActiveDismissed',
```

Comment:
Looks like this was intended to be used in `Applets.tsx` but is no longer referenced and has been replaced by a unique key generated by `getDismissedKey`. Can this be removed?